verticaltabs.jsm

```diff
@@ -465,6 +500,47 @@ VerticalTabs.prototype = {
     });
   },
 
+  clearFind: function () {
+    this.document.getElementById('find-input').value = '';
+    this.filtertabs();
+  },
+
+  filtertabs: function () {
+    let document = this.document;
+    let tabs = this.document.getElementById('tabbrowser-tabs');
+    let find_input = this.document.getElementById('find-input');
+    let input_value = find_input.value.toLowerCase();
+    let hidden_counter = 0;
+    let filterable_tabs = tabs.querySelectorAll('tab[verticaltabs-id]');
```

Comment:
(As a side note, if we remove the VTTabDataStore.jsm, then we'll need to change this selector…)